web/src/containers/StateDashboard.js

```diff
@@ -0,0 +1,101 @@
+import PropType from 'prop-types';
+import React from 'react';
+import { connect } from 'react-redux';
+
+import Icon, { faPlusCircle, faSpinner } from '../components/Icons';
+import Sidebar from './StateDashboardSidebar';
+import TopBtns from './TopBtns';
+import { createApd, selectApd } from '../actions/apd';
+import { Section } from '../components/Section';
+import { t } from '../i18n';
+
+const Loading = () => (
+  <div className="h2 p0 pb3 center">
+    <Icon icon={faSpinner} spin size="sm" className="mr1" /> Loading APDs
+  </div>
+);
+
+const StateDashboard = ({
+  apds,
+  createApd: create,
+  fetching,
+  selectApd: select,
+  state
+}) => {
+  const open = id => e => {
+    e.preventDefault();
+    select(id);
+  };
+
+  return (
+    <div className="site-body">
+      <Sidebar />
+      <div className="site-main p2 sm-p4 md-px0">
+        <TopBtns hideDashboard />
+
+        <Section resource="stateDashboard">
+          <div className="mb3 bg-white rounded shadow accordian">
+            <div className="px3 py2 border-bottom border-bottom-darken-1 blue">
+              <button
+                className="btn bg-blue white rounded p1 right"
+                size="small"
+                onClick={create}
+              >
+                Create new&nbsp;&nbsp;
+                <Icon icon={faPlusCircle} />
+              </button>
+              <span className="h2">{state.name} APDs</span>
```

Comment:
Any chance we could swap the order of the span and the button? The DOM order here means a screen reader will read "Create New" and then "<state> APDs," and it might make more sense the other way around. Also, can we make the span an `h2` element so it's officially the section header here? (Might have to tell it to `display: inline` or `display: inline-block` to get it to not break the button on its own line.)